tests/test_segyview.py

```diff
@@ -0,0 +1,53 @@
+from unittest import TestCase
+import segyio
+import segyview.util as util
+import itertools
+from test_segy import TestSegy
+
+
+class TestSegyView(TestCase):
```

Comment:
These tests should just read the `test-data/small.sgy` and use the `SegyIO API` to compare between memory and file.